mod/01_rating/set/abstract/01_metric_child.rb

```diff
@@ -0,0 +1,109 @@
+# include with option :generation
+# @example
+#   include_set Abstract::MetricChild, generation: 2
+#   # means the includer is grandchild (e.g. name is like metric+a+b)
+
```

Comment:
reduces code repetition in Type::MetricValue and TypePlusRight::MetricValue::Value